spec/controller_services/games_controller/update_service_spec.rb

```diff
@@ -32,8 +32,8 @@
     context 'when the params are invalid' do
       let!(:game) { create(:game) }
```

Comment:
I don't know why Rubocop doesn't think this one needs to be aligned with the others but I feel like if I change it it'll break.